ej2-react/gantt/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Gantt component | Syncfusion
-description:  Checkout and learn about Getting started with React Gantt component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with React Gantt Component | Syncfusion
+description: Checkout and learn about getting started with React Gantt component of Syncfusion Essential JS 2 and more details.
 control: Getting started 
 platform: ej2-react
 documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Getting started
+# Getting Started React Gantt Component
```

Comment:
Change like this - Getting Started with React Gantt Component

Reply:
changed